### ISSUE TITLE
File not found dist/index.js error when trying to run action

### ISSUE THREAD

**Issue by daniestevez** [katyo/publish-crates#701]
I'm trying to run the action from the current `main` branch, rather than the v2 release, since #648 should be fixed in `main`.

I'm getting the following error:
```
Run katyo/publish-crates@9766ec7ed8ebbb0d34168814ecacf51833b6f97d
Error: File not found: '/home/runner/work/_actions/katyo/publish-crates/9766ec7ed8ebbb0d34168814ecacf51833b6f97d/dist/index.js'
```

See the [job run](https://github.com/maia-sdr/maia-sdr/actions/runs/8017155027/job/21900543038).

**Comment by katyo** [comment]
@daniestevez Because I don't like keep derived products with source code, you should not use such branches in CI. Instead use `v*` tags or `dist/v*` branches. Keep in mind this branches and tags updates automatically when CI run complete successfully.

**Comment by daniestevez** [comment]
Does the `dist/v2` branch contain the fix for #648?

**Comment by katyo** [comment]
@daniestevez The `dist/v2` actually derived from `main` with by adding build products.
You can see commit log to be sure.
